src/disco/containers/InstallButton.js

```diff
@@ -143,9 +152,11 @@ export function mapDispatchToProps(dispatch) {
       });
     },
 
-    uninstall({ guid, installURL, slug }) {
+    uninstall({ guid, installURL, slug, name, type }) {
       const addonManager = new AddonManager(guid, installURL);
       dispatch({type: 'START_UNINSTALL', payload: {slug}});
+      const action = type === THEME_TYPE ? 'theme' : 'addon';
+      tracking.sendEvent({action, category: UNINSTALL_CATEGORY, label: name});
```

Comment:
We could, we might do it differently in the future. Disco needs to be done with certain strings for continuity we'd probably want something different for a new amo.